src/utils/calldata.ts

```diff
@@ -0,0 +1,250 @@
+import BN from 'bn.js';
```

Comment:
I have one nitpick, since this is now a new entry in the `utils`, can you please also add the function definitions in the utils documentation pages?

Reply:
Done !